common/dbutils/suffix_type.go

```diff
@@ -1,70 +1,65 @@
 package dbutils
 
-import "encoding/binary"
+import (
+	"bytes"
+	"github.com/ledgerwatch/turbo-geth/rlp"
+)
 
-type Suffix []byte
-
-func ToSuffix(b []byte) Suffix {
-	return Suffix(b)
+func Decode(b []byte) (SuffixHistory, error) {
+	if len(b) == 0 {
+		return SuffixHistory{
+			Changes: make([]Change, 0),
+		}, nil
+	}
+	buf := bytes.NewBuffer(b)
+	h := SuffixHistory{}
+	err := rlp.Decode(buf, &h)
```

Comment:
I would prefer not to use `rlp` package, if possible. It is very inefficient both in terms of allocations and cpu usage. That is why I used hand-written encoding before